apps/gro/controllers/confirm.js

```diff
@@ -27,32 +28,16 @@ ConfirmController.prototype.saveValues = function saveValues(req, res, callback)
 
     var subject = locali18n.translate('pages.email.information.subject');
 
-    var d = {
-      values: (function getSessionValues(data) {
-        var r = {};
-        for (var prop in data) {
-          r[prop] = data[prop];
-        }
-        return r;
-      }(req.sessionModel.attributes))
-    };
+    var d = {values: _.pick(req.sessionModel.attributes, _.identity)};
```

Comment:
Could we not just deep clone the object? `const data = Object.assign({}, req.sessionModel.attributes);`